contracts/container/config.yml

```diff
@@ -9,7 +9,7 @@ events:
       - name: containerID
         type: Hash256
       - name: publicKey
-        type: PublicKey
+        type: ByteArray
```

Comment:
Can this change be avoided?

Reply:
i tried passing N3 verification and invocation scripts as public key and signature parameters correspondingly. Verification script was 47 byte len. Faced following error:
```
System.Runtime.Notify failed: notification PutSuccess is invalid: parameter 1 type mismatch: PublicKey (manifest) vs ByteString (notification)
```
so i decided that this extension is necessary. Is there a better way to get around this with min changes to the contract?